code/modules/mining/equipment/explorer_gear.dm

```diff
@@ -114,3 +114,61 @@
 		var/mutable_appearance/M = mutable_appearance('icons/mob/head.dmi', "hostile_env_glass")
 		M.appearance_flags = RESET_COLOR
 		. += M
+
+
+// CITADEL ADDITIONS BELOW
+
+/****************SEVA Suit and Mask****************/
+
+/obj/item/clothing/suit/hooded/seva
```

Comment:
you'll need to add 
`mutantrace_variation = NO_MUTANTRACE_VARIATION` unless making Digitigrade versions of these